code/modules/mob/mob_defines.dm

```diff
@@ -48,7 +48,8 @@
 	*/
 
 	/// The zone this mob is currently targeting
-	var/zone_selected = BODY_ZONE_CHEST
+	/// Use select_bodyzone and get_combat_bodyzone to get this value
+	VAR_PRIVATE/zone_selected = BODY_ZONE_CHEST
```

Comment:
Remove the privateness before merge imo, its just ugly

Reply:
VAR_PRIVATE prevents people from directly accessing the variable outside of the proc, which is required for this case since you shouldn't be directly accessing this and instead should be using the helpers. The reason I did this wasn't for me, but for future PRs/ports which need to fail the checks if they have not been setup correctly. Var private is very important here and shouldn't change (VS code compilation will give you warnings in the editor with this setup too).